<?xml version="1.0" encoding="UTF-8"?>
<root BTCPP_format="4">
  <BehaviorTree ID="Main_BT">
    <Sequence>
      <client_test/>
      <action_client_test/>
    </Sequence>
  </BehaviorTree>

  <!-- Description of Node Models (used by Groot) -->
  <TreeNodesModel>
    <Action ID="action_client_test"
            editable="true"/>
    <Action ID="client_test"
            editable="true"/>
  </TreeNodesModel>

</root>
